portfolio/src/main/webapp/index.html

```diff
@@ -8,11 +8,93 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-    </div>
+
+      
+      <h1 id = "header">My Portfolio</h1>
+
+      
+      <div id="aboutMe">
+	<h3>About Me: a short history</h3>
+	<p class= "iBox">
+	  My name is Mauricio. I was born in San Miguel, El Salvador 1998. I moved to the states, thanks to my dad, at age 7. I grew up in northern Virginia less than an hour from DC.
+	  I was a curious little kid, first I figured out how to speak english, then progressed with the childhood thing. In middle school the tech-ed teacher said 'hey wanna build robots
+	  after school and stuff?' and that sounded dope so I started doing that. In highschool I liked history and physics the most but math more than english.
+	  I got really  into robotics and engineering in general, it seemed right up my alley. I love problem solving, puzzles, and building things. I was particularly keen on programming!
+	  So I got myself into an engineering school and decided to pursue a computer science degree, so here I am living the dream... as my family's designated tech guy! Whooo! 
+	</p>
+      </div>
+      
+
+      <div id="infoBox">
+	<img src="/images/img1.jpeg" id="self" class= "sPic">
```

Comment:
All the images are broken, but there's an easy fix! It should be "images/..." instead of "/images/..." for all of them!